packages/default/scss/grid/_layout.scss

```diff
@@ -900,6 +900,13 @@
         font-size: inherit;
     }
 
+    .k-pager-input {
+        .k-numerictextbox {
+            margin: 0 1ex;
```

Comment:
I think we should check if there is any variable for the numerictextbox which is responsible for the font-size/height of the element. If there is not, is it a good idea to introduce a variable?